crates/hc_sandbox/src/passphrase.rs

```diff
@@ -0,0 +1,86 @@
+use once_cell::sync::Lazy;
```

Comment:
I have the sense that this file is copy pasted, because I also copy pasted it over from the `holochain` binary over to `holochain-runner` at one point, and I think *it* in turn got copy pasted from lair-keystore code? 
Maybe this can be avoided somehow, is this a useful utility?

Reply:
sorry just realized this is in draft. I was just checking out PRs and we had touched base about this recently

Reply:
@Connoropolous - yeah, I'd like to expose it from lair, but we're in a bit of a freeze over there due to the security audit...

Reply:
Oh gotcha. Good to know!

Reply:
@Connoropolous - Okay, moved into `holochain_util` under feature flag `pw` - we can pull it further into lair at a later date and holochain_util can be a re-export.